llvm/lib/Target/AMDGPU/SIOptimizeExecMasking.cpp

```diff
@@ -313,9 +313,6 @@ MachineBasicBlock::reverse_iterator SIOptimizeExecMasking::findExecCopy(
   return E;
 }
 
-// XXX - Seems LivePhysRegs doesn't work correctly since it will incorrectly
-// report the register as unavailable because a super-register with a lane mask
-// is unavailable.
```

Comment:
From the context, I'm not sure the comment should be dropped. Maybe just should refer to LiveRegUnits